tui2/pages/progress/progress.go

```diff
@@ -105,4 +105,5 @@ func (p *Progress) SetSize(w, h int) {
 	headerHeight := 7
 	p.Common.SetSize(w, h)
 	p.bars.SetSize(w, h-headerHeight)
+	p.Styles.StatusBarValue.Width(p.Common.Width - len("Parallel engine progress messages: ") - 2)	// adjust width to force word wrap
```

Comment:
To avoid magic numbers I've added a helper function to calculate labels pane width in case labels change in the future.
If you prefer to just hardcode the number at 36 let me know.